### ISSUE TITLE
[Urinary bladder] Unmapped CL terms

### ISSUE THREAD

**Issue by anitacaron** [obophenotype/asct-b_validation#59]
From the report listing CT terms without IDS: _[2022-10-17 Release](https://github.com/hubmapconsortium/ccf-validation-tools/blob/v2022-10-17/logs/new_cl_terms_Urinary_bladder.tsv)_

Some of the CL names in the report already exist in Cell Ontology:

| Terminal AS/ID | Terminal AS/label                 | Terminal AS/user_label            | CL Name                                | CL term    | OLS link                                                                                              |
|----------------|-----------------------------------|-----------------------------------|----------------------------------------|------------|-------------------------------------------------------------------------------------------------------|
| UBERON:0004645 | urinary bladder urothelium        | urinary bladder urothelium        | basal cell of bladder urothelium       | CL:1000486 | https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_1000486 |
| UBERON:0001261 | lamina propria of urinary bladder | lamina propria of urinary bladder | myofibroblast cell of bladder          | CL:0000186 | https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000186 |
| UBERON:0012239 | urinary bladder vasculature       | urinary bladder vasculature       | endothelial cell of bladder            | CL:0000115 | https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000115 |
| UBERON:0012239 | urinary bladder vasculature       | urinary bladder vasculature       | vascular smooth muscle cell of bladder | CL:0000359 | https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000359 |
| UBERON:0012239 | urinary bladder vasculature       | urinary bladder vasculature       | pericyte cell of bladder               | CL:0000669 | https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000669 |

**Comment by emquardokus** [comment]
@anitacaron this whole table is in the process of a major rework by author; will revisit for April 2023 CCF release
They have determined differences in the different regions of the urinary tract, so not sure we should use generic cell ontology terms for basal cell of urothelium for the entire urinary tract. Likewise [myofibroblast cell](CL_0000186), generic endothelial cell CL:0000115 where we know there are differences and heterogeity. 
Basically, these were purposefully not mapped to generic cell types by author.

**Comment by anitacaron** [comment]
@emquardokus I suggested changing to a generic cell type to ensure that we can bulk create using this approach explained in https://github.com/hubmapconsortium/ccf-ontology/issues/16
